fighthealthinsurance/common_view_logic.py

```diff
@@ -1619,7 +1625,7 @@ async def generate_appeals(cls, parameters) -> AsyncIterator[str]:
         async for dt in denial.denial_type.all():
             form = await sync_to_async(dt.get_form)()
             if form is not None:
-                parsed = form(parameters)
+                parsed= form(parameters)
```

Comment:
this is inconsistent style and will be changed next time we run black so probably better to run black first.